plugins/joblib_launcher/example/my_app.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import time
+
+import hydra
+
+
+@hydra.main(config_path="conf/config.yaml")
+def my_app(cfg):
+    print(f"Executing task {cfg.task} ...")
```

Comment:
1. Use log.info()
2. Log process ID as well (assuming JobID runs in multiple processes).